test/rubygems/test_gem_installer.rb

```diff
@@ -1142,14 +1142,24 @@ def test_install_extension_dir_is_removed_on_reinstall
   end
 
   def test_find_lib_file_after_install
```

Comment:
What mean `two separate tests`?  test_find_lib_file_after_install and test_install_extension_flat ?